docs/learn/encyclopedia/contract-development/types/fully-typed-contracts.mdx

```diff
@@ -69,29 +66,54 @@ Options:
   -h, --help  Print help
 ```
 
-Like any other CLI, you can also get help for any of these subcommands using something like `native balance --help`. Stellar CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples.
+Like any other CLI, you can also get help for any of these subcommands using something like `… -- balance --help`. Stellar CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples.
```

Comment:
Is it normal to have the "..." here? It might be! Just caught my eye.

Reply:
The full command this is v long, and I'm abbreviating it here. If that's confusing, I can explain it explicitly. Maybe:

```suggestion
Like any other CLI, you can also get help for any of these subcommands. Omitting everything before the `--` in the previous command, this would look like: `… -- balance --help`. Stellar CLI again fetches the on-chain interface types, this time using it to generate a full list of all arguments to the function, and even generates examples.
```